index.Dockerfile

```diff
@@ -33,8 +35,8 @@ COPY --from=builder --chown=1001:0 /bin/grpc_health_probe /bin/grpc_health_probe
 
```

Comment:
Let's remove microdnf update since it's not only updating the existing packages, but is also installing new packages and that results increases the image size and violates the 'minimum size possible' guideline from the IBM certification.